c2rust-transpile/src/translator/assembly.rs

```diff
@@ -186,7 +314,8 @@ fn reg_is_reserved(constraint: &str) -> Option<(&str, &str)> {
 ///
 /// Modifies operands and returns a pair of prefix and suffix strings that
 /// should be appended to the assembly template.
-fn rewrite_reserved_reg_operands(att_syntax: bool, operands: &mut Vec<BidirAsmOperand>) -> (String, String) {
+fn rewrite_reserved_reg_operands(att_syntax: bool, arch: Arch,
```

Comment:
Run rustfmt? This doesn't look like how it would break this line.

Reply:
Done.